src/aktualizr_lite/CMakeLists.txt

```diff
@@ -0,0 +1,25 @@
+if(BUILD_OSTREE)
+add_executable(aktualizr-lite main.cc)
+target_link_libraries(aktualizr-lite aktualizr_static_lib ${AKTUALIZR_EXTERNAL_LIBS})
+
+install(TARGETS aktualizr-lite RUNTIME DESTINATION bin COMPONENT aktualizr-lite)
+
+set(TEST_SOURCES test_lite.sh)
+
+add_custom_target(t_aktualizr-lite)
+add_dependencies(t_aktualizr-lite aktualizr-repo)
+add_dependencies(t_aktualizr-lite aktualizr-lite)
+add_dependencies(build_tests t_aktualizr-lite)
```

Comment:
This does appear to work now. However, I'm not sure that you need a custom target at all. You should just need `add_dependencies(build_tests aktualizr-lite)`.

Reply:
Good point. I was trying to make it really easy to just run "make t_aktualizr-lite" and generate just what you need for unit testing. However, "make build_tests" is fine. Fixed now.